src/Analyser/NodeScopeResolver.php

```diff
@@ -363,6 +367,7 @@ private function processNode(\PhpParser\Node $node, Scope $scope, \Closure $node
 					$scopeClass = $argValueType->getReferencedClasses()[0];
 				} elseif (
 					$argValue instanceof Expr\ClassConstFetch
+					&& is_string($argValue->name)
```

Comment:
ClassConstFetch::$name can never be `Error`:

```
Error nodes will not be present if the parser is run in throwOnError mode (the default).
```

I know phpDoc says differently but I'd rather have a way of overriding this.

Reply:
Agreed